README.md

```diff
@@ -20,127 +19,273 @@ let thirdNumber = numbers[2]
 
 ```swift
  if let index = names.indexOf("Jim") {
     names.removeAtIndex(index)
  }
- 
+
  // ["Sarah", "Bran", "Sansa"]
 ```
 
-* Referring to `count`, they should be exposed to the following where it is explained thoroughly what is going on: 
+- Referring to `count`, they should be exposed to the following where it is explained thoroughly what is going on:
 
 ```swift
 let rainbowColors = ["Red", "Orange", "Yellow", "Green", "Blue", "Indigo", "Violet"]
- 
+
  let numberOfColors = rainbowColors.count
  // 7
- 
+
  print("There are \(numberOfColors) colors in the rainbow.")
  // prints "There are 7 colors in the rainbow.
- 
+
  if rainbowColors.count > 5 {
     print("There are more than 5 colors in a rainbow.")
  }
  //prints "There are more than 5 colors in a rainbow."
 ```
 
-* Referring to `isEmpty` - I like the idea of stepping them through the following two examples:
+- Referring to `isEmpty` - I like the idea of stepping them through the following two examples:
 
 ![deliLine](http://i.imgur.com/ivgaSon.png?1)
+
 ```swift
  var deliLine = [
     "Albert Einstein",
     "Isaac Newton",
     "Galileo Galilei",
     "Marie Curie"
  ]
- 
+
  if deliLine.isEmpty {
     print("The deli line is empty")
  } else {
     print("There are people waiting to eat!")
  }
- 
+
  // prints "There are people waiting to eat!"
 ```
+
 ![Moons](http://i.imgur.com/reNsnn6.png?1)
+
 ```swift
  let moonsOfJupiter = [
     "Europa",
     "Ganymedge",
     "Io",
     "Callisto"
 ]
- 
+
  if !moonsOfJupiter.isEmpty {
     for moon in moonsOfJupiter {
         print("\(moon) is a moon orbiting Jupiter.")
     }
  }
- 
+
  // Europa is a moon orbiting Jupiter.
  // Ganymede is a moon orbiting Jupiter.
  // Io is a moon orbiting Jupiter.
  // Callisto is a moon orbiting Jupiter.
 ```
 
+--------------------------------------------------------------------------------
+
+In the last lesson you briefly saw a 'method', which can be thought of as a pre-written function that helps you accomplish common tasks. They are contextual, in that, for example, a method that works on an array may not work for a String.
+
+The method you saw in the last example was `.append()` that adds a value to the end of an array. You could achieve this in several ways which would involve far more code and calculation, I think you'll agree that `.append()` is far more convenient. There are many other methods for arrays, and in this lesson you'll look at some others.
+
+## Enumeration
+
+Before going any further, there's one other array concept to introduce.
+
```

Comment:
If we go through with my suggestion below, it would make sense to include the `toMakeTea` variable here.

```swift
var toMakeTea = [
    "Boil Water",
    "Add tea bag to cup",
    "Wait ten minutes",
    "Add Milk",
    "Drink"
]
```